include/arch/arm/aarch32/irq.h

```diff
@@ -7,7 +7,7 @@
 
 /**
  * @file
- * @brief Cortex-M public interrupt handling
+ * @brief ARM public interrupt handling
```

Comment:
Can we name this AARCH32?

Reply:
or ARM 32-bit

Reply:
Thanks for reviewing. I will change this to `ARM AArch32` to be consistent with `exc.h`.